libbz2-rs-sys/src/decompress.rs

```diff
@@ -1285,8 +1278,6 @@ pub(crate) fn decompress(
 
                     EOB = s.nInUse + 1;
                     nblockMAX100k = s.blockSize100k as u8;
-                    groupNo = -1;
-                    groupPos = 0;
```

Comment:
moved over to hopefully make it clearer why `groupNo` cannot be unsigned (currently anyway, maybe with some extra refactoring).